module-2/backend/week_one.md

```diff
@@ -1,34 +1,111 @@
 ## Week One - Module 2 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!). 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!).
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+  get - This is used to read a record
+  post - Used to create a record
+  put - Used to update a record
+  patch - Also used to update a record??

```

Comment:
Yup! PUT/PATCH are generally used interchangeably, though PUT is technically to fully replace a resource while PATCH is to update some attributes.